code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -61,7 +63,34 @@
 			name = "improvised explosive"
 			active = 0
 			det_time = rand(30,80)
+	else
+		if(assembled == 2 && current_shrapnel < max_shrapnel )
+			if(I.is_shrapnel && I.w_class == W_CLASS_TINY || I.w_class == W_CLASS_TINY || I.is_shrapnel)
```

Comment:
And: `&&`
Or: `||`

I have no idea what exactly you're trying to do here, but the `I.is_shrapnel && I.w_class == W_CLASS_TINY` is currently completely extraneous because one of the other checks is guaranteed to pass if that one does.
Did you mean to check if both vars are true, or only if either is?